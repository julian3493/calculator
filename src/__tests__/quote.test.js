import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../pages/quote';

it('it test the render Home page', () => {
  const quote = renderer
    .create(<Quote />)
    .toJSON();
  expect(quote).toMatchSnapshot();
});
