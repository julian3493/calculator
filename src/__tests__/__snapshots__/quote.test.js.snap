// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it test the render Home page 1`] = `
<div
  className="container"
>
  <p
    className="quote"
  >
    Mathematics is not about number, equations, computations or algorithms: is about understading.
    <span>
       - William Paul Thurston.
    </span>
  </p>
</div>
`;
